### PR TITLE
fix(chat): improve session sidebar stats readability

### PR DESCRIPTION
**Pull request by alexinthesky**
## Summary
- Split the cramped single-line session meta row (date, message count, and run stats mixing `•`/`·` separators) into two rows: date/message-count on one line, run stats on their own line below.
- Run stats now use a single consistent `·` separator, small `@grafana/ui` `Icon`s (`bolt` for tokens, `repeat` for turns, `cog` for tool calls — matching the `cog` used elsewhere for tool calls) to cut text density, and abbreviated token counts (`47015` → `47k`, `1000000` → `1.0M`) with the exact count preserved in a hover tooltip.
- No changes to `SessionStats`/`getSessionStats` fetching logic — this is purely a rendering change in `SessionItem`. The existing zero-runs / not-yet-loaded graceful-degradation behavior is unchanged.

## Test plan
- [x] `npm run typecheck` passes
- [x] `npm run lint` passes (only pre-existing, unrelated deprecation warnings)
- [x] `npm run test:ci` passes (504/504), including updated `SessionSidebar.test.tsx` coverage for the new `formatTokenCount` abbreviation logic
- [x] Updated `tests/sessionManagement.spec.ts` regex to match the new abbreviated/labeled stats text
- [x] Manually verified in a local Grafana dev environment (`npm run server`): sent a real chat message and confirmed the sidebar renders `⚡ 28k tokens · ↻ 1 turn · ⚙ 1 tool call` on its own row with a `28,214 tokens` tooltip

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Low Risk**
> Pure UI/rendering change in the session sidebar with no auth, data, or fetching logic changes.
> 
> **Overview**
> Improves chat history sidebar readability by splitting the cramped session meta into **two rows** (date/messages, then run stats) and abbreviating token counts (`47k`, `1.0M`) via a new `formatTokenCount` helper, with the exact count kept in a hover tooltip.
> 
> Run stats now use consistent `·` separators plus small icons (`bolt`, `repeat`, `cog`). Stats fetching is unchanged — this is display-only. Unit and E2E assertions are updated for the new format.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 24df80009a4812047623f4c8c03cd52177375bf3. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: src/components/Chat/components/SessionSidebar/SessionSidebar.tsx [hunk 1]
```diff
@@ -310,6 +310,19 @@ function ShareDialogWrapper({
...
+function formatTokenCount(tokens: number): string {
+  if (tokens < 1000) {
+    return tokens.toString();
+  }
+  if (tokens < 10000) {
+    return `${(tokens / 1000).toFixed(1)}k`;
+  }
+  if (tokens < 1000000) {
+    return `${Math.round(tokens / 1000)}k`;
+  }
```
[the remark this answers is not in the document]
Comment (alexinthesky) [reply src/components/Chat/components/SessionSidebar/SessionSidebar.tsx:324]: Fixed — `formatTokenCount` now checks the rounded value and promotes to the "M" suffix when rounding would otherwise reach 1000k (e.g. 999,600 → "1.0M" instead of "1000k"). Added a regression test for the boundary case.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
